packages/pages/src/vite-plugin/build/buildStart/rendering/templateUtils.ts

```diff
@@ -59,7 +59,6 @@ export interface PluginRenderTemplates {
 export const getPluginRenderTemplates = async (
   manifest: Manifest
 ): Promise<PluginRenderTemplates> => {
-  const clientRenderPath = pathLib.join("..", manifest.renderPaths._client);
   const serverRenderPath = manifest.renderPaths._server.replace("assets", "..");
```

Comment:
hm, what is the reasoning something needs to be done in the client path?

Reply:
Is that because in the server case it is relative to the filesystem?

Reply:
yes